dependencies/pixi-filters.js

```diff
@@ -1,9 +1,9 @@
 /*!
```

Comment:
I'm not sure we even use any of the extra pixi filters... But we totally could.

I'll make a note to check what v4 has added over v3